homeassistant/components/yolink/services.yaml

```diff
@@ -0,0 +1,37 @@
+siren_turn_on:
```

Comment:
Why do you need a service for this? Eventually I would map on/off to a switch entity.